### ISSUE TITLE
Cannot create AccessContextManagerAccessLevel sample, invalid osConstraints format

### ISSUE THREAD

**Issue by AlexBulankou** [GoogleCloudPlatform/k8s-config-connector#155]
**Describe the bug**
Cannot create AccessContextManagerAccessLevel sample, validation failure list:
spec.basic.conditions.devicePolicy.osConstraints in body must be of type array: "object"

**ConfigConnector Version**
1.7.2

**To Reproduce**
Run samples/resources/accesscontextmanageraccesslevel/accesscontextmanager_v1beta1_accesscontextmanageraccesslevel.yaml sample from the installation package.

Error: validation failure list:
spec.basic.conditions.devicePolicy.osConstraints in body must be of type array: "object"

**Comment by caieo** [comment]
There's a small error in the sample YAML (osConstraints needs to list the osTypes as objects (the '-' was the only thing missing)). A fix will be out in the next release. 

Here is what spec.basic.conditions.devicePolicy.osConstraints should look like in the spec:
```
spec:
  basic:
    conditions:
      - devicePolicy:
          osConstraints:
          - osType: DESKTOP_CHROME_OS
```

**Comment by jcanseco** [comment]
Hi @AlexBulankou. This sample has been fixed in the `v1.8.0` release. I'll be closing this issue now.
